### PR TITLE
Re-adds the recipes for Kelotane, Bicardine, Antitoxin, and Tricordazine.

### PR DESCRIPTION
**Pull request by TechnoAlchemisto**
The tricord recipe is a tweaked a bit, and they all have OD thresholds courtesy of goofball's forward thinking.
:cl:
rscadd: The recipes for some Trekchems are now back in the game
rscadd: Bicaridine can be made with carbon, oxygen, and sugar.
rscadd: Kelotane can be made with silicon and carbon.
rscadd: Antitoxin can be made with nitrogen, silicon, and potassium
rscadd: tricordrazine can be made by combining all three.
/:cl:

### CODE REVIEW

**Comment by Bawhoppen** [comment]
:+1:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:x:100:

**Comment by Supermichael777** [comment]
at this point yout just cutting open old wounds for goof to pour more salt into

**Comment by Cheridan** [comment]
sorry, no references to copyrighted material allowed

**Comment by Shapsy** [comment]
Antitoxin is strictly worse than charcoal I think so may as well leave it out

**Comment by Bawhoppen** [comment]
SHAPS DON'T YOU DARE FUCKING CRITICIZE THIS

**Comment by PKPenguin321** [comment]
Beat me to it tbh

**Comment by Shapsy** [comment]
FINE HAVE YOUR PISSWATER MEME CHEMICAL

**Comment by Cheridan** [comment]
Joking aside it would be good if we took the opportunity to rename it to dylovene, which is what antitoxin was referred to in some places. Sounds less generic.

**Comment by TechnoAlchemisto** [comment]
You can cut charcoal and antitox together just FYI

**Comment by Zilenan91** [comment]
For what purpose?

**Comment by Eaglendia** [comment]
The prospect of injecting charcoal into your veins to cure toxins still offends me in ways that no one seems to understand

**Comment by ShadowDeath6** [comment]
Dylovene was generally used for making non-ODing omnizine patches, which charcoal is less effective at due to purging other chemicals. (Citation needed)

**Comment by Jazaen** [comment]
Well, you can throw charcoal together with antitox, for MAX toxin healing (though counterproductive, since one purges another)

**Comment by Hatfish** [comment]
Why don't you just up anti-toxin's values and remove charcoal's toxin healing altogether?

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 1]
```diff
@@ -220,3 +220,31 @@
...
+/datum/chemical_reaction/kelotane
+	name = "Kelotane"
+	id = "kelotane"
+	result = "kelotane"
+	required_reagents = list("carbon" = 1, "silicon" = 1)
+	result_amount = 2
+
+
+/datum/chemical_reaction/antitoxin
+	name = "Antitoxin"
```
Comment (xxalpha) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: It's 'Dylovene', correct it.

**Comment by xxalpha** [comment]
Tricordrazine, Dylovene, Bicaridine.

At least use the correct names.

**Comment by TechnoAlchemisto** [comment]
Dylovene is not its name in the code, but the other names have been corrected.

**Comment by xxalpha** [comment]
The old object name used to be "Dylovene (Anti-toxin)" I think.

**Comment by TechnoAlchemisto** [comment]
yeah but the actual reagent name is "antitoxin" if this somehow gets merged maybe I'll put in the effort to switch the name out

**Comment by GunHog** [comment]
**HEAVY BREATHING**

**Comment by lzimann** [comment]
:^)

**Comment by duncathan** [comment]
no merge unless you name it Dylovene

**Comment by WJohn** [comment]
Can we remove the goof meds that do this but are touch only, then? This is super redundant to have both.

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: honestly leave it as anti-toxin

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 2]
```diff
@@ -220,3 +220,31 @@
...
+	id = "antitoxin"
+	result = "antitoxin"
+	required_reagents = list("nitrogen" = 1, "silicon" = 1, "potassium" = 1)
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: doublecheck that these dont conflict with anything please

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 3]
```diff
@@ -220,3 +220,31 @@
...
+	id = "bicaridine"
+	result = "bicaridine"
+	required_reagents = list("carbon" = 1, "oxygen" = 1, "sugar" = 1)
+	result_amount = 3
+
+/datum/chemical_reaction/kelotane
+	name = "Kelotane"
+	id = "kelotane"
+	result = "kelotane"
+	required_reagents = list("carbon" = 1, "silicon" = 1)
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: doublecheck that these dont conflict with anything please

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 4]
```diff
@@ -220,3 +220,31 @@
 	required_reagents = list("chlorine" = 1, "fluorine" = 1, "aluminium" = 1, "potass_iodide" = 1, "oil" = 1)
 	result_amount = 5
 
+/datum/chemical_reaction/bicardine
+	name = "Bicaridine"
+	id = "bicaridine"
+	result = "bicaridine"
+	required_reagents = list("carbon" = 1, "oxygen" = 1, "sugar" = 1)
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: doublecheck that these dont conflict with anything please

**Comment by Iamgoofball** [comment]
@WJohn fuck off, the touch onlys actually heal more if you apply them properly and have actual uniqueish mechanics

:+1: to this, I'm fine with these memes

just dont remove the other chems, keep anti-tox and charcoal in

**Comment by WJohn** [comment]
There is no justification for their existence or even touch based chems. The only unique mechanic there is that they are poisonous if eaten or injected rather than applied. Injectable chems work perfectly normally when used as smoke or foam, that should be considered touch.

**Comment by Iamgoofball** [comment]
And that's a reason to remove it because...?

We don't lose anything from keeping it.

**Comment by WJohn** [comment]
When you have something like 400 chems, useless identical bloat that can easily be trimmed is something you should consider doing. I don't mind multiple methods to do the same thing, there's probably 6 or 7 ways to create slabs of meat and that's fine, because they're all from different places and achieved differently. Two identical chems that are going to be made in a chem dispenser 99% of the time does not justify the simple existence of the 1% where someone somehow gets to chose only one of the two through ghetto means.

Either make the chems much more significantly different in healing mechanics, or make it something you can reliably produce outside of chemistry.

**Comment by Sarcalogo** [comment]
What about Dermaline?

**Comment by duncathan** [comment]
touch chems are more restricted in application but heal better
these chems are easy to apply but less effective

they each have their niche and we're not gonna remove them

**Comment by TechnoAlchemisto** [comment]
Alrighty now that THAT'S fixed. It is done.

This is mostly just to give botany bees more healing option, ability to make healing grenades again, and to make the cool healing cigarettes. Also more opportunities to use pills again.

**Comment by Zilenan91** [comment]
Can you buff doctor's delight with this too? These chems are really easy to create so DD will be left kinda in the dust. Maybe just remove the hunger drain from it if you don't want to buff the stats.

**Comment by TechnoAlchemisto** [comment]
This PR will be purely for readding the named chemicals, further tweaks can be addressed later.

**Comment by Iamgoofball** [comment]
Oh my god the literal slippery slope I stated 6 months ago is happening

Never ever tell me I'm wrong on a slippery slope again

**Comment by newfren** [comment]
The recipes for these are hilariously simple, please nerf their healing slightly to compensate for how easy it is to make them.

**Comment by TechnoAlchemisto** [comment]
I assure you that goof's healing chems are still superior to these, their ease of creation is balanced by the fact that they aren't better and they have a low OD threshold.

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: NEEDSMOREDYLOVENE

**Comment by PKPenguin321** [comment]
@TechnoAlchemisto Go update the chemistry wiki page now

**Pull request merged**